.travis.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       script: sudo ./tools/travis/run_tests.sh snapcraft/tests/integration/general
     - if: type != cron
       script: sudo ./tools/travis/run_tests.sh snapcraft/tests/integration/plugins
+    - if: type != cron
+      script: sudo ./tools/travis/run_tests.sh snapcraft/tests/integration/plugins/catkin
```

Comment:
I'm afraid specifying `-t` to discover means this must be importable, which of course it is not (see the Travis error).

Using -t seems unnecessary, things work fine without it, and then our plan actually works. I'm going to push that change up and see how things run.